core/src/main/java/com/tweet/core/controller/UserController.java

```diff
@@ -22,9 +22,9 @@ public ResponseEntity createUser(User user) {
         return ResponseEntity.ok().build();
     }
 
-    @GetMapping(value = "/{username}")
```

Comment:
We may still be needing get by user name, but time will tell. I will start working on UI this weekend and than it will become clear. Its pretty hard to come up with controller without UI so no worries now